docs/standard/components.md

```diff
@@ -37,9 +33,9 @@ There are four primary .NET implementations that Microsoft actively develops and
 
 ### .NET Core
 
-.NET Core is a cross-platform implementation of .NET optimized for server workloads. It's modern, efficient, and designed to handle server and cloud workloads at scale. It implements the .NET Standard, so code that targets the .NET Standard can run on .NET Core. ASP.NET Core runs on .NET Core.
+.NET Core is a cross-platform implementation of .NET optimized for server workloads. It runs on Windows, macOS and Linux. It's modern, efficient, and designed to handle server and cloud workloads at scale. It implements the .NET Standard, so code that targets the .NET Standard can run on .NET Core. ASP.NET Core runs on .NET Core. 
```

Comment:
Perhaps delete the "It's modern, efficient..." and change the first sentence to end 'for server and cloud workloads"?